patternlab/styleguide/source/_patterns/01-atoms/09-media/figure--iframe.twig

```diff
@@ -0,0 +1,15 @@
+{% if figure.iframe %}
+  {# Render figure.content, which will be the main content of the figure #}
+  {# Important: we wrap this in an `if` statement to allow consuming systems
+     to either use figure.image or figure.content (if they prefer to externally
+     render figure.content
+   #}
+  {% set figure = figure|merge({
+    'content': include("@atoms/09-media/iframe.twig", {'iframe': figure.iframe})
```

Comment:
Ah, I just tested this on the Drupal side, and the function form of `include` doesn't work (the markup gets escaped).  In my suggestions PR, I just converted it to the block form for `figure--iframe` and `figure--image`.

Reply:
Implemented in #971.